Iceberg-Plugin-GitHub.package/IceGitHubNewPullRequestAction.class/instance/basicExecute.st

```diff
@@ -3,22 +3,25 @@ basicExecute
 
 	credentials := IceGitHubAPI ensureCredentials.
 
-	remote ifNil: [  
+	remote ifNil: [ 
 		remote := self repository branch hasUpstream
 			ifTrue: [ self repository branch upstream remote ]
-			ifFalse: [ self askRemote: self repository ].
+			ifFalse: [ self askRemote: self repository ]
 	].
 
 	(self validateMakePullRequestOn: self repository) ifFalse: [ ^ #() ].
-		
+
 	(IceGitHubCreatePullRequestModel 
-		repository: self repository 
+		repository: self repository
 		credentials: self credentials
 		headRemote: remote)
-		setModal: true; 
+		setModal: true;
 		onAccept: [ :pullRequest | 
-			[ 
-				pullRequest send.
-				UIManager default inform: 'Pull request created.' ]
-			on: IceGitHubError do: [ :e | self reportError: e ] ];
-		openWithSpec 
\ No newline at end of file
+			[ | pullRequestDatas |
+			pullRequestDatas := pullRequest send.
+			UIManager default 
+				inform: 'Pull request created. Click to view on Github.' 
+				actionOnClick: [ self class environment at: #WebBrowser ifPresent: [ :webBrowser | webBrowser openOn: (pullRequestDatas at: 'html_url') ] ifAbsent: [ self inform: 'Feature not available in Pharo 6' ]  ] ]
```

Comment:
Same comment as in #1048, I'd like a more explicit comment. (Also this shows that we are repeating some code here...)